test/api_tests.jl

```diff
@@ -13,12 +13,11 @@ mutable struct DiscreteVelocity <: AbstractAgent
     vel::NTuple{2, Int}
     diameter::Float64
 end
-mutable struct AmbiguousAgent <: AbstractAgent
-    id::Int
-    pos::NTuple{2, Int}
-    # The following types are abstract
-    weight::Real
-    info::AbstractString
+mutable struct ParametricAgent{T} <: AbstractAgent where {T <:Integer}
```

Comment:
```suggestion
mutable struct ParametricAgent{T <: Integer} <: AbstractAgent
```
Is the correct syntax. `where` is reserved for method definitions :)

Reply:
Oh, yes&mdash;thanks. Updated.